content/python/concepts/classes/classes.md

```diff
@@ -6,51 +6,58 @@ Subjects:
   - 'Data Science'
   - 'Web Development'
 Tags:
+  - 'Attributes'
   - 'Classes'
+  - 'Methods'
   - 'Objects'
   - 'OOP'
 CatalogContent:
   - 'learn-python-3'
   - 'paths/computer-science'
 ---
 
-**Classes** are templates used to define the properties and methods of objects in code. They can describe the kinds of information that the class holds, and also how a programmer interacts with that data.
+**Classes** are templates used to define the properties and methods of objects in code. They can describe the kinds of data that the class holds, and also how a programmer interacts with that data.
 
 The usage of classes are a key element of [object-oriented programming](https://www.codecademy.com/resources/docs/general/object-oriented-programming) (OOP) where "class instances" are created to implement design principles such as [inheritance](https://www.codecademy.com/resources/docs/general/inheritance) and [encapsulation](https://www.codecademy.com/resources/docs/general/encapsulation).
 
 ## Creating a Class
 
-In Python, classes are defined using the `class` keyword.
-
-```py
-class Animal:
-  def __init__(self, name, leg_count):
-    self.name = name
-    self.leg_count = leg_count
-```
-
-Class definitions cannot be empty, so the `pass` statement can be used as a placeholder to avoid errors.
-
 ```py
 class Home:
-  pass
+  # Class body starts here
 ```
 
-## Creating Instances of a Class
+In Python, classes are defined using the [`class`](https://www.codecademy.com/resources/docs/python/keywords/class) keyword. Within the code of the class, we can call its methods and access its variables. We do this by using `self` followed by a period `.` and then followed by the method or variable.
 
-Objects can be created from classes. These objects are called instances of a class, and when we create an instance, that is known as instantiating a class.
+### Class Instances
 
-To create an instance of a class, we can set a variable equal to the class name followed by parentheses `()`.
+Objects can be created or instantiated, from classes. These objects are known as class instances. To create an instance of a class, we can set a variable equal to the class name followed by parentheses `()`:
 
 ```py
 my_home = Home()
 ```
 
 Here, the instance name is `my_home`, and the class is `Home`.
 
-## Class Methods
+### Attributes
+
+Class attributes are variables that are defined outside of all methods and have the same value for every instance of the class.
```

Comment:
```suggestion
Class attributes are variables that are defined outside of all methods and have the same value for every instance of the class. They also can be accessed via the class name rather than the instance name. Setting the variable via the class name will change it for all instances. **Note:** setting it via an instance name will break the connection for that instance.
```

Reply:
You don't have to use this language, but some discussion of the idiosyncrasies of python class attributes is warranted.